Metrica_Viz.py

```diff
@@ -545,30 +557,43 @@ def plot_pitchcontrol_for_event(
     if plotting_difference:
         PPCF = convert_pitch_control_for_cmap(PPCF)
 
+    reds_cmap = 'Reds'
+    blues_cmap = 'Blues'
+    home_cmap = 'bwr'
+    away_cmap = 'bwr_r'
+
+
+    if len(cmap_list):
+        blues_cmap = LinearSegmentedColormap.from_list("", cmap_list[:6][::-1])
+        reds_cmap = LinearSegmentedColormap.from_list("", cmap_list[6:])
+        home_cmap = LinearSegmentedColormap.from_list("", cmap_list)
+        away_cmap = LinearSegmentedColormap.from_list("", cmap_list[::-1])
+
```

Comment:
If we are adding functionality to change the colors to something other than red or blue, perhaps we should call these variables something other than  "reds_cmap, blues_cmap"

Reply:
changed to home/away_presence_cmap